FAIMS/FAIMS3
# feat: jump to end

## JIRA Ticket

[BSS-390](https://jira.csiro.au/browse/BSS-390)

## Description

Add a button that allows the user to scroll directly to the bottom of the record form.

## Proposed Changes

- created transparent button component
- created function to scroll to divs
- added button to both new and edit record screens

## How to Test

1. Navigate to a survey
2. Create a new record
3. Ensure the "Jump to End" button is above the form and test its function
4. (optional) Publish the record
5. (optional) Open the saved record and repeat step 3

## Additional Information

## Checklist
